documentation/index.md

```diff
@@ -38,11 +40,19 @@ workshop websites:
 
 #### This website
 
-The source for this website is located on github [here](https://github.com/boutproject/boutproject.github.io). If you would like to
-contribute to these pages please contact [one of the developers](mailto:benjamin.dudson@york.ac.uk).
-Most of the pages are written using [Markdown](http://daringfireball.net/projects/markdown/syntax).
+The source for this website is located on github [here](websitegithub). Pull
```

Comment:
Should the syntax use round brackets like here, or square as a few lines below? Sorry, showing my md ignorance